packages/compass-import-export/src/components/export-json-format-options.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+  Accordion,
+  Link,
+  Radio,
+  RadioGroup,
+  css,
+  spacing,
+} from '@mongodb-js/compass-components';
+
+import type { ExportJSONFormat } from '../export/export-json';
+
+const radioGroupStyles = css({
+  margin: `${spacing[3]}px 0`,
+});
+
+function JSONFileTypeOptions({
+  jsonFormat,
+  setJSONFormatVariant,
+}: {
+  jsonFormat: ExportJSONFormat;
+  setJSONFormatVariant: (jsonFormatVariant: ExportJSONFormat) => void;
+}) {
+  return (
+    <Accordion
+      text="Advanced JSON Format"
+      data-testid="export-advanced-json-format"
+    >
+      <RadioGroup
+        className={radioGroupStyles}
+        data-testid="export-json-format-options"
+        onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+          setJSONFormatVariant(event.target.value as ExportJSONFormat)
+        }
+      >
+        <Radio
+          value="default"
+          checked={jsonFormat === 'default'}
+          description='Example:  { "fortyTwo": 42, "oneHalf": 0.5, "bignumber": { "$numberLong": "5000000000" } }'
```

Comment:
I think it would be a nice subtle hint (and bit if detail) if you could use a number > 2^^32. Then for default and canonical it would be correct and relaxed it would be wrong. ie. if you use 123456789123456784, then relaxed would turn that into 123456789123456780.